ocaml/idl/datamodel_host.ml

```diff
@@ -1291,6 +1291,15 @@ let create_params =
     ; param_release= dundee_release
     ; param_default= Some (VDateTime Date.epoch)
     }
+  ; {
+      param_type= String
+    ; param_name= "last_update_hash"
+    ; param_doc=
+        "The SHA256 checksum of updateinfo of the most recently applied update \
+         on the host"
+    ; param_release= numbered_release "2.21.0-next"
```

Comment:
We are at version 24.39.0 already.

Reply:
I see in "numbered_release", the parameter "version" is passed to “get_product_releases”: https://github.com/xapi-project/xen-api/blob/master/ocaml/idl/datamodel_common.ml#L379
In "get_product_releases", it goes through "release_order": https://github.com/xapi-project/xen-api/blob/master/ocaml/idl/datamodel_types.ml#L360, which is the XenServer releases.
So I think the string here should be "XenServerVersion-next", not the "XapiVersion-next"?

Reply:
We don't use XS version here anymore, just xapi versions. It's the same as for the lifecycle fields, but the type is a little different.

Reply:
OK, I see.